src/components/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * This function is responsible for rendering the left content column of the screen.
+ * It is also responsible for handling the credential creation process
+ * @returns The `ColumnLeftWrap` component returns a `div` element with a `div`
+ * element inside of it.
+ */
+import { useState } from 'react'
+import SchemasManager from '../SchemasManager'
+import DefinitionsManager from '../DefinitionsManager'
+import AttributesManager from '../AttributesManager'
+import IssueAndResetButtons from '../IssueAndResetButtons/IssueAndResetButtons'
+import usePostIssueCredentialSendOffer from '../../../interface/hooks/use-post-issue-credential-send-offer'
+
+export default function ColumnLeftWrap({ origin, persona }) {
```

Comment:
is persona still relevant? @jonmattgray I can't remember what we did on the authority one?

Reply:
we completely removed `persona` in `veritable-authority`. Wherever `persona` is eventually used can be replaced by the`ISSUER_LABEL` constant in this repo

Reply:
@jonmattgray it's already done in 

```js
import { ISSUER_LABEL } from './utils/env'

function App() {
  return <AppCore agent={ISSUER_LABEL} />
}
```

Reply:
and in `AppCore.js`

```js
 const persona = `${ISSUER_LABEL}.agent`
```

Reply:
The string passed on to persona is used in places like `usePostCredentialDefinitions` when creating the schema prefix like

```js
const schemaDefTagPrefix = `${persona}.agent`
```

you can just pull persona from the global ISSUER_LABLE from there .. 

i don't mind , from my point of view either way is good..